.bazelrc

```diff
@@ -24,3 +24,6 @@ test --build_tests_only
 # as the user configuration should be able to overwrite flags from this file.
 # See https://docs.bazel.build/versions/master/best-practices.html#bazelrc
 try-import %workspace%/user.bazelrc
+
+# C++17 is required for the MCAP support in ROS Bag
+build --cxxopt='-std=c++17'
```

Comment:
Please import https://github.com/mvukov/rules_ros2/blob/main/ros2/cc_defs.bzl#L7 and use where needed instead.